### PR TITLE
feat: German language Part 1

### PR DESCRIPTION
**Pull request by mprangenberg**
This is the first pull request containing changes to commands to support multiple languages. I also translated those to german along the way.

### CODE REVIEW

File: commands/resource.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const { YZEmbed } = require('../utils/embeds');
 const YZRoll = require('../yearzero/YZRoll');
 const { emojifyRoll } = require('../Sebedius');
 
-const ARTIFACT_DIE_REGEX = /^d(6|8|10|12)$/i;
+const ARTIFACT_DIE_REGEX = /^[dw](6|8|10|12)$/i;
```
Comment (Stefouch) [inline commands/resource.js:5]: Why this change ?

File: commands/roll.js [hunk 2]
```diff
@@ -17,66 +17,10 @@ const DICE_RANGE_ICONS = {
 
 module.exports = {
 	name: 'roll',
-	aliases: ['r', 'lance', 'lancer', 'slå', 'sla'],
+	aliases: ['r', 'lance', 'lancer', 'slå', 'sla', 'würfeln', 'wurf'],
```
Comment (Stefouch) [inline commands/roll.js:20]: I think it would be best to remove any translation in the command aliases array (french and swedish too) or it'll become crowded after just a few supported languages.

File: commands/roll.js [hunk 3]
```diff
@@ -116,16 +64,16 @@ module.exports = {
 		else game = await ctx.bot.getGame(ctx, 'myz');
 
 		// Creates the roll.
-		let roll = new YZRoll(game, ctx.author, name);
+		let roll = new YZRoll(game, ctx.author, name, lang);
 
 		// Year Zero Roll Regular Expression.
-		const yzRollRegex = /^((\d{1,2}[dbsgna])|([bsgna]\d{1,2})|(d(6|8|10|12))|([abcd])+)+$/i;
+		const yzRollRegex = /^((\d{1,2}[dbsfgnaw])|([bsfgna]\d{1,2})|([dw](6|8|10|12))|([abcd])+)+$/i;
```
Comment (Stefouch) [inline commands/roll.js:70]: I am not supporting a change of the regex based on language alternatives for "d".
Please, revert back to original this line and the other.

File: gamedata/fbl/demon-generator.de.json [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+				"symbol": "👻"
+			}
+		}
+	},
+	"abilities": {
+		"roll": "D66",
+		"array": true,
+		"data": {
+			"11": ["Steinhaut", "Rüstungsschutz +6"],
+			"13": ["Riesig", "Stärke +2W6"],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:111]:
```suggestion
			"13": ["Riesig", "Stärke +2D6"],
```

File: gamedata/fbl/demon-generator.de.json [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+			"15": ["Brennender Körper", "Immun gegen Feuer."],
+			"21": ["Aus Eis", "Rüstungsschutz +2"],
+			"23": ["Mumifiziert", "Furchtangriff W6+5"],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:114]:
```suggestion
			"23": ["Mumifiziert", "Furchtangriff D6+5"],
```

File: gamedata/fbl/demon-generator.de.json [hunk 6]
```diff
@@ -0,0 +1,286 @@
...
+			"25": ["Schleimig", "Kann sich durch sehr kleine Öffnungen bewegen."],
+			"31": ["Übernatürlich schön", "Manipulation +2"],
+			"33": ["Von Geschwüren bedeckt", "Ansteckende Berührung, Ansteckungsgefahr W6+5"],
+			"35": ["Federn", "Rüstungsschutz +2"],
+			"41": ["Flügel", "Kann fliegen, Bewegungsrate 3"],
+			"44": ["Von Licht erfüllt", "Furchtangriff W6+5"],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:120]:
```suggestion
			"44": ["Von Licht erfüllt", "Furchtangriff D6+5"],
```

File: gamedata/fbl/demon-generator.de.json [hunk 7]
```diff
@@ -0,0 +1,286 @@
...
+		"_comment": "The official german version has a bad translation on '26'. They translated 'turns' with 'Runden' (rounds) instead of 'Züge'. As that doesn't make sense the entry differs here.",
+		"data": {
+			"11": ["Immun gegen Waffen", "Nur waffenlose Angriffe und Zauber können den Dämon verletzen."],
+			"15": ["Immun gegen Feuer", "Kann nicht von Feuer verletzt werden."],
+			"23": ["Immun gegen Kälte", "Kälte hat keinen Einfluss auf den Dämon."],
+			"26": ["Blitzschnell", "Zieht zwei Initiativekarten und handelt in beiden Zügen."],
+			"33": ["Immun gegen physische Angriffe", "Physische Angriffe sind wirkungslos."],
+			"35": ["Parasit", "Der Dämon kann Kontrolle über sein Opfer ergreifen, indem er es berührt. Dies funktioniert wie der Zauber PUPPENSPIELER mit Machtstufe 3."],
+			"41": ["Regeneration", "Erhält jede Runde W3 Stärke zurück."],
+			"44": ["Lähmung", "Ein Opfer, das vom Dämon berührt wird, leidet unter einem lähmenden Gift mit Wirksamkeit W6+5."],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:250]:
```suggestion
			"44": ["Lähmung", "Ein Opfer, das vom Dämon berührt wird, leidet unter einem lähmenden Gift mit Wirksamkeit D6+5."],
```

File: gamedata/fbl/demon-generator.de.json [hunk 8]
```diff
@@ -0,0 +1,286 @@
...
+		"array": true,
+		"_comment": "The official german version has a bad translation on '26'. They translated 'turns' with 'Runden' (rounds) instead of 'Züge'. As that doesn't make sense the entry differs here.",
+		"data": {
+			"11": ["Immun gegen Waffen", "Nur waffenlose Angriffe und Zauber können den Dämon verletzen."],
+			"15": ["Immun gegen Feuer", "Kann nicht von Feuer verletzt werden."],
+			"23": ["Immun gegen Kälte", "Kälte hat keinen Einfluss auf den Dämon."],
+			"26": ["Blitzschnell", "Zieht zwei Initiativekarten und handelt in beiden Zügen."],
+			"33": ["Immun gegen physische Angriffe", "Physische Angriffe sind wirkungslos."],
+			"35": ["Parasit", "Der Dämon kann Kontrolle über sein Opfer ergreifen, indem er es berührt. Dies funktioniert wie der Zauber PUPPENSPIELER mit Machtstufe 3."],
+			"41": ["Regeneration", "Erhält jede Runde W3 Stärke zurück."],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:249]:
```suggestion
			"41": ["Regeneration", "Erhält jede Runde D3 Stärke zurück."],
```

File: gamedata/fbl/demon-generator.de.json [hunk 9]
```diff
@@ -0,0 +1,286 @@
...
+			"44": ["Lähmung", "Ein Opfer, das vom Dämon berührt wird, leidet unter einem lähmenden Gift mit Wirksamkeit W6+5."],
+			"46": ["Giftig", "Ein Opfer, das vom Dämon berührt wird, leidet unter einem tödlichen Gift mit Wirksamkeit W6+5."],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:251]:
```suggestion
			"46": ["Giftig", "Ein Opfer, das vom Dämon berührt wird, leidet unter einem tödlichen Gift mit Wirksamkeit D6+5."],
```

File: gamedata/fbl/demon-generator.de.json [hunk 10]
```diff
@@ -0,0 +1,286 @@
...
+		}
+	},
+	"weaknesses": {
+		"roll": "D66",
+		"array": true,
+		"data": {
+			"11": ["Wasser", "Erleidet W3 Schaden, wenn er mit Wasser angespritzt wird."],
+			"14": ["Feuer", "Erleidet doppelten Schaden durch Feuer."],
+			"16": ["Kälte", "Erleidet doppelten Schaden durch Kälte."],
+			"22": ["Licht", "Lichtempfindlich; muss sich von starken Lichtquellen wegbewegen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden von kleineren Lichtquellen (z. B. Fackeln) und W6 Schaden von großen Lichtquellen (z. B. Sonne)."],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:266]:
```suggestion
			"22": ["Licht", "Lichtempfindlich; muss sich von starken Lichtquellen wegbewegen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden von kleineren Lichtquellen (z. B. Fackeln) und D6 Schaden von großen Lichtquellen (z. B. Sonne)."],
```

File: gamedata/fbl/demon-generator.de.json [hunk 11]
```diff
@@ -0,0 +1,286 @@
...
+			"25": ["Silber", "Erleidet doppelten Schaden durch Waffen aus Silber."],
+			"32": ["Gold", "Erleidet doppelten Schaden durch Waffen aus Gold."],
+			"34": ["Kinder", "Kann den Anblick von Kindern nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden."],
+			"36": ["Elfen", "Kann den Anblick von Elfen nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden."],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:270]:
```suggestion
			"36": ["Elfen", "Kann den Anblick von Elfen nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde D3 Schaden."],
```

File: gamedata/fbl/demon-generator.de.json [hunk 12]
```diff
@@ -0,0 +1,286 @@
...
+		"roll": "D66",
+		"array": true,
+		"data": {
+			"11": ["Wasser", "Erleidet W3 Schaden, wenn er mit Wasser angespritzt wird."],
+			"14": ["Feuer", "Erleidet doppelten Schaden durch Feuer."],
+			"16": ["Kälte", "Erleidet doppelten Schaden durch Kälte."],
+			"22": ["Licht", "Lichtempfindlich; muss sich von starken Lichtquellen wegbewegen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden von kleineren Lichtquellen (z. B. Fackeln) und W6 Schaden von großen Lichtquellen (z. B. Sonne)."],
+			"25": ["Silber", "Erleidet doppelten Schaden durch Waffen aus Silber."],
+			"32": ["Gold", "Erleidet doppelten Schaden durch Waffen aus Gold."],
+			"34": ["Kinder", "Kann den Anblick von Kindern nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden."],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:269]:
```suggestion
			"34": ["Kinder", "Kann den Anblick von Kindern nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde D3 Schaden."],
```

File: gamedata/fbl/demon-generator.de.json [hunk 13]
```diff
@@ -0,0 +1,286 @@
...
+			"36": ["Elfen", "Kann den Anblick von Elfen nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden."],
+			"42": ["Zwerge", "Kann den Anblick von Zwergen nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden."],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:271]:
```suggestion
			"42": ["Zwerge", "Kann den Anblick von Zwergen nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde D3 Schaden."],
```

File: gamedata/fbl/demon-generator.de.json [hunk 14]
```diff
@@ -0,0 +1,286 @@
...
+			"44": ["Heilige Symbole", "Kann den Anblick von heiligen Symbolen nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden."],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:272]:
```suggestion
			"44": ["Heilige Symbole", "Kann den Anblick von heiligen Symbolen nicht ertragen und muss sich von ihnen entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde D3 Schaden."],
```

File: gamedata/fbl/demon-generator.de.json [hunk 15]
```diff
@@ -0,0 +1,286 @@
...
+			"53": ["Musik", "Kann den Klang von Musik und Melodien nicht ertragen und muss sich davon entfernen. Falls dies nicht möglich ist, erleidet der Dämon jede Runde W3 Schaden."],
+			"61": ["Erde", "Erleidet W3 Schaden durch den Kontakt mit Erde."],
```
Comment (Stefouch) [inline gamedata/fbl/demon-generator.de.json:274]:
```suggestion
			"61": ["Erde", "Erleidet D3 Schaden durch den Kontakt mit Erde."],
```

File: gamedata/fbl/fbl-magic-mishaps.de.csv [hunk 16]
```diff
@@ -0,0 +1,18 @@
+ref;effect
+11-13;Jemand beobachtet deine Magie und erzählt anderen davon. Deine Reputation steigt um einen Punkt.
+14-15;Deine Magie macht dich sehr `HUNGRIG`.
+16-21;Dein Zauber macht dich auf einmal sehr `DURSTIG`.
+22-23;Die Magie stört deinen Schlaf. Du kannst W6 Tage lang nicht schlafen, wodurch du `MÜDE` wirst.
+24-25;Der Zauber nimmt dir die Energie und fügt 1 Punkt Schaden auf Geschicklichkeit zu.
+26-31;Deine Magie schadet deinem Körper. Du erleidest 1 Punkt Schaden auf Stärke.
+32-33;Die Macht deiner eigenen Magie überwältigt dich und lässt die Menschen in deiner Umgebung neben dir wie Ameisen wirken. Du erleidest 1 Punkt Schaden auf Empathie.
+34-35;Der Zauber entfesselt dämonische Visionen, die dir 1 Punkt Schaden auf Verstand zufügen.
+36-41;Der Zauber löst eine magische Krankheit mit einer Ansteckungskraft von 2W6 aus. Du und alle auf `ARMESLÄNGE` sind für den nächsten Tagesabschnitt der Ansteckung ausgesetzt.
```
Comment (Stefouch) [inline gamedata/fbl/fbl-magic-mishaps.de.csv:10]:
```suggestion
36-41;Der Zauber löst eine magische Krankheit mit einer Ansteckungskraft von 2D6 aus. Du und alle auf `ARMESLÄNGE` sind für den nächsten Tagesabschnitt der Ansteckung ausgesetzt.
```

File: gamedata/myz/myz-monsters-catalog.en.csv [hunk 17]
```diff
@@ -1,5 +1,5 @@
 id;strength;agility;wits;empathy;speed;hp;armor;skills;special;source;attacks
-mmyz-humanoid-amnesiac;2;2;4;2;1;1;0;Comprehend 2|Heal 1;;myz;{unarmed}
+mmyz-humanoid-amnesiac;2;2;4;2;1;1;0;Comprehend 2|Heal 1;;myz;{wmyz-unarmed}
```
Comment (Stefouch) [inline gamedata/myz/myz-monsters-catalog.en.csv:2]: This change must be double-checked

File: gamedata/myz/myz-weapons-catalog.en.csv [hunk 18]
```diff
@@ -48,7 +48,7 @@ wmyz-scrapderringer;1;1;1;1;true;Light;Jury-Rigged;myz
 wmyz-scrapflamethrower;2;1;2;1;true;Fire|Heavy;Jury-Rigged;myz
 wmyz-scrapknife;1;1;2;0;;Light;Jury-Rigged;myz
 wmyz-scrapmachete;1;2;2;0;;;Jury-Rigged;myz
-wmyz-scrappistol;1;2;2;2;true;;Jury-Rigged;myz
+wmyz-scrappistol;1;1;2;2;true;;Jury-Rigged;myz
```
Comment (Stefouch) [inline gamedata/myz/myz-weapons-catalog.en.csv:51]:
```suggestion
wmyz-scrappistol;1;2;2;2;true;;Jury-Rigged;myz
```
Are you sure of this change?
In version 4 it is 2 damage for the scrap pistol.

File: utils/Util.js [hunk 19]
```diff
@@ -684,6 +696,19 @@ class Util {
...
 	}
+
+	/**
+	 * Takes the provided language code, checks it against the SUPPORTED_LANGS-table 
+	 * and if not found calls the getLanguage-method to read from DB or return default
+	 * @param {string} lang Language code (for example provided by arguments)
+	 * @param {ContextMessage} ctx The context (for bot and guild.id)
+	 * @returns {string} A valid language code
+	 */
+	static async getValidLanguageCode(lang, ctx){
```
Comment (Stefouch) [inline utils/Util.js:707]: I would better see this method in the Sebedius class but the change can wait for a later update.

File: utils/constants.js [hunk 20]
```diff
@@ -2,9 +2,11 @@ module.exports = {
 	SUPPORTED_GAMES: ['myz', 'fbl', 'tales', 'coriolis', 'alien', 'vaesen', 't2k'],
 	SUPPORTED_LANGS: {
 		en: 'English',
+		de: 'Deutsch',
 		// fr: 'Français',
 		// sw: 'Svenska',
 	},
+	LOWERCASE_LANGS: ['en'],
```
Comment (Stefouch) [inline utils/constants.js:9]: Why not ?
```js
Object.keys(SUPPORTED_LANGS)
```
instead of `LOWERCASE_LANGS` ?

File: utils/locales.js [hunk 21]
```diff
@@ -1,8 +1,346 @@
+const { SUPPORTED_GAMES } = require('./constants');
+const Config = require('../config.json');
+
 const LOCALES = {
 	en: {
```
Comment (Stefouch) [inline utils/locales.js:5]: For a further release, I will separate each language in its own file.

**Review by Stefouch — CHANGES_REQUESTED** [review]
Hi @mprangenberg !
Impressive work. You did something I couldn't start since I was afraid by the size of the task, but you basically made the bot translatable everywhere.

Please see comments. There are a few things that I need to discuss.

File: commands/resource.js (hunk 1 above)
Comment (mprangenberg) [reply commands/resource.js:5]: This is because i noticed the most common mistake while playing with germans is, that they are used to do common dice rolls as i.e. 1w6, where w stands for the german word for dice "Würfel". I wanted to make it easier for my players :D

File: commands/roll.js (hunk 2 above)
Comment (mprangenberg) [reply commands/roll.js:20]: Yeah, i wanted to make it easier for my players, but i see your point.

File: commands/roll.js (hunk 3 above)
Comment (mprangenberg) [reply commands/roll.js:70]: Alright, will do.

File: gamedata/myz/myz-weapons-catalog.en.csv (hunk 18 above)
Comment (mprangenberg) [reply gamedata/myz/myz-weapons-catalog.en.csv:51]: I made this change based on the current german PDF versions of the core book and mechatron, which usually already contain errata from english versions, and the free starter booklet from drivethrurpg. My google fu couldn't find anything about this. And the german Mechatron book is from 2020.

File: gamedata/myz/myz-monsters-catalog.en.csv (hunk 17 above)
Comment (mprangenberg) [reply gamedata/myz/myz-monsters-catalog.en.csv:2]: {unarmed} would just say "Unarmed". And {wmyz-unarmed} would cover that for attacks, wouldn't it?

File: utils/constants.js (hunk 20 above)
Comment (mprangenberg) [reply utils/constants.js:9]: Maybe it is a bad name? This is because german isn't a language which is all lowercase, unlike english. So i had to distinguish this for the legend generator (see FBLLegendGenerator).

File: utils/locales.js (hunk 21 above)
Comment (mprangenberg) [reply utils/locales.js:5]: OK, let me know how you want to do that and if you need any help.

File: commands/resource.js (hunk 1 above)
Comment (mprangenberg) [reply commands/resource.js]: I reverted the commit

File: commands/roll.js (hunk 2 above)
Comment (mprangenberg) [reply commands/roll.js]: I pushed a new commit where i removed the language aliases from commands, other than the english language one

File: commands/roll.js (hunk 3 above)
Comment (mprangenberg) [reply commands/roll.js]: Reverted that commit

File: gamedata/fbl/demon-generator.de.json (hunk 4 above)
Comment (mprangenberg) [reply gamedata/fbl/demon-generator.de.json]: I changed all the dice references to support the english "D"-format. I think you're right and this is best to have one abbreviation for all.

**Comment by mprangenberg** [comment]
I'll try to go through the remaining commands over the next few weekends.
Also: i spoke with the german publisher of all systems (except alien and tales) and they said they are ok with us using the official translations and texts from the books. Alien and tales are under another publisher (because of reasons), which i already contacted.

File: gamedata/myz/myz-weapons-catalog.en.csv (hunk 18 above)
Comment (mprangenberg) [reply gamedata/myz/myz-weapons-catalog.en.csv:51]: Oh, i just saw this. I didn't change the damage, that's two in my versions, too. I changed the bonus, which says +1 in my books.

File: utils/Util.js (hunk 19 above)
Comment (mprangenberg) [reply utils/Util.js:707]: I can move the method there if you'd like. The question is, if i should just change the "getLanguage" method to support an optional language code argument or put them side by side, as they maybe accomplish different things.

**Comment by mprangenberg** [comment]
Ok, i just got the go ahead from the second publisher. They are really happy and are going to announce it on their Discord when it's available.

File: gamedata/myz/myz-weapons-catalog.en.csv (hunk 18 above)
Comment (Stefouch) [reply gamedata/myz/myz-weapons-catalog.en.csv:51]: > Scrap Pistol: +1 Gear Die, 2 damage
This is the stats for the Scrap Pistol in the v4 of the English PDF (2 dec 2018).
Is it the same for the German version ?

File: utils/Util.js (hunk 19 above)
Comment (Stefouch) [reply utils/Util.js:707]: Let's keep two different methods!

File: gamedata/myz/myz-monsters-catalog.en.csv (hunk 17 above)
Comment (Stefouch) [reply gamedata/myz/myz-monsters-catalog.en.csv:2]: You are right, but the syntax `{something}` also returns a default weapon, which is... an unarmed attack.. but with range!

So yes, your correction is better and fixes an issue I didn't see.

See: https://github.com/Stefouch/sebedius-myz-discord-bot/blob/fae188c804cbc79a493fa3a3691506367f974b47/yearzero/YZObject.js#L343

File: utils/locales.js (hunk 21 above)
Comment (Stefouch) [reply utils/locales.js:5]: I'll do it after this pull request is merged.
I will also propose you to join the project as a collaborator.

File: gamedata/myz/myz-weapons-catalog.en.csv (hunk 18 above)
Comment (mprangenberg) [reply gamedata/myz/myz-weapons-catalog.en.csv:51]: Yes, it's the same. +1 Bonus (gear die), 2 Damage. In your proposed version it's +2 Bonus and 2 Damage.

File: utils/Util.js (hunk 19 above)
Comment (mprangenberg) [reply utils/Util.js:707]: Alright, i will make the changes and push them propably later today.

File: utils/locales.js (hunk 21 above)
Comment (mprangenberg) [reply utils/locales.js:5]: Thank you, i'm happy to join the team :)

File: gamedata/myz/myz-weapons-catalog.en.csv (hunk 18 above)
Comment (Stefouch) [reply gamedata/myz/myz-weapons-catalog.en.csv:51]: Oh yes! I forgot that the 2nd value is for the `grip`.

File: gamedata/fbl/fbl-journeys.de.yml [hunk 22]
```diff
@@ -0,0 +1,423 @@
...
+  id: JOURNEY_MISHAP_FORAGE
+  name: Sammeln
+  roll: D6
+  refs:
+    '1':
+      - Giftig
+      - >-
+        Du findest eine Einheit `GEMÜSE`, aber sie ist giftig – was euch aber
+        erst bei eurer nächsten Mahlzeit auffällt. Das Gift in dem `GEMÜSE`
+        hat eine__ Wirksamkeit von **3**__.
```
Comment (Stefouch) [inline gamedata/fbl/fbl-journeys.de.yml:127]:
```suggestion
        hat eine __Wirksamkeit von **3**__.
```

**Review by Stefouch — COMMENTED** [review]
Almost ready

File: utils/constants.js (hunk 20 above)
Comment (mprangenberg) [reply utils/constants.js]: I changed the constant to KEEP_CAPITALIZATION_LANGS (with a comment) to better represent what it does: https://github.com/Stefouch/sebedius-myz-discord-bot/pull/81/commits/1855484b2b696389836693fd49f85970b5eaefa0

**Review by Stefouch — APPROVED** [review]
